src/mlm_server_engine.inc

```diff
@@ -1447,10 +1447,10 @@ s_server_config_service (s_server_t *self)
         if (streq (zconfig_name (section), "security")) {
             char *mechanism = zconfig_get (section, "mechanism", "null");
             char *domain = zconfig_get (section, "domain", NULL);
-            if (domain)
```

Comment:
Could you please keep this one in place manually for now? We need to sort the whole thing out and then fix in zproto eventually